React/game/src/app/components/authRoute/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { PureComponent } from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+class ValidateRouter extends PureComponent {
+  renderRoute = props => {
+    const { isPrivate, component: Component, isLogin } = this.props;
+    if (isLogin && !isPrivate) {
+      return <Redirect to="/game" />;
```

Comment:
Here, the redirection will always take you to `/game` even if you try to access another route (like the other ones requested on the top bar). No need to change anything now, but keep this in mind for future tasks